prow/config.yaml

```diff
@@ -20726,18 +20742,18 @@ periodics:
       args:
       - |-
         --query=is:pr
-        -labels:do-not-merge
-        -labels:do-not-merge/blocked-paths
-        -labels:do-not-merge/cherry-pick-not-approved
-        -labels:do-not-merge/hold
-        -labels:do-not-merge/release-note-label-needed
-        -labels:do-not-merge/work-in-progress
-        labels:lgtm
-        labels:approved
+        -label:do-not-merge
+        -label:do-not-merge/blocked-paths
+        -label:do-not-merge/cherry-pick-not-approved
+        -label:do-not-merge/hold
+        -label:do-not-merge/release-note-label-needed
+        -label:do-not-merge/work-in-progress
+        label:lgtm
+        label:approved
         status:failure
-        -labels:needs-rebase
-        -labels:needs-ok-to-test
-        -labels:"cncf-cla: no"
+        -label:needs-rebase
+        -label:needs-ok-to-test
+        -label:"cncf-cla: no"
```

Comment:
(also shrug for those - not sure if there are more instances :-( )

Reply:
what's going on here?

Reply:
oh :(((

Reply:
hummm, assume you did something like `s/-label/-labels` to this file? I was diff the job yaml for this job and this was changed

Reply:
yeah :/
I'll go back over the previous diff to check for more of these.